networking/synthetic/stac-test.yaml

```diff
@@ -8,15 +8,40 @@
     
   - name: Copy Dockerfile to producer and consumer
     copy: src=stac/Dockerfile dest=/tmp/stac/Dockerfile
+
+  - name: Copy onload rpms to producer and consumer
+    copy:
+      src: "{{ item }}"
+      dest: "/tmp/stac/"
+    with_fileglob:
+      - "stac/openonload-*.rpm" 
+
+  - name: Create stac tuned profile directory if not exists
+    file: path=/etc/tuned/stac state=directory
     
   - name: Copy tuned stac profile to producer and consumer
+    copy: src=stac/tuned.conf dest=/etc/tuned/stac/tuned.conf
+
+  - name: Copy tuned stac profile to docker image
     copy: src=stac/tuned.conf dest=/tmp/stac/tuned.conf
 
+  - name: Copy stac profile script to producer and consumer
+    copy: src=scripts/tuning.sh dest=/etc/tuned/stac/tuning.sh
+
+  - name: Update interface name in tuned script
+    shell: sed -i -- "s/p5p1/{{ iface_name }}/" /etc/tuned/stac/tuning.sh
```

Comment:
there is an ansible "replace" module or a "lineinfile" module to deal with this natively